evm/upgradable.js

```diff
@@ -42,6 +42,26 @@ async function deployCreateUpgradable(
     return new Contract(proxy.address, implementationJson.abi, wallet);
 }
 
+async function deployCreate2(
```

Comment:
This should be present in a different file since this one is for upgradable contracts, perhaps in `deploy/utils.js`

Reply:
Instead of having separate files for upgradable and non-upgradable deployment scripts, why not move them all into deploy/utils.js?